charts/opentelemetry-operator/values.yaml

```diff
@@ -11,6 +11,9 @@ nameOverride: ""
 ## Reference one or more secrets to be used when pulling images from authenticated repositories.
 imagePullSecrets: []
 
+## Kubernetes cluster domain suffix
+clusterDomain: cluster.local
```

Comment:
I think we should nest this under `certManager` unless such a setting would be useful other places in the chart.

Reply:
I've found lots of charts have this as clusterDomain or global.clusterDomain, so it's less surprising there